Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/UI/DriverOnboarding/DriverLicense.hs

```diff
@@ -146,9 +149,9 @@ verifyDL isDashboard mbMerchant personId req@DriverDLReq {..} = do
 verifyDLFlow :: Person.Person -> Text -> UTCTime -> Id Image.Image -> Maybe (Id Image.Image) -> Maybe UTCTime -> Flow ()
 verifyDLFlow person dlNumber driverDateOfBirth imageId1 imageId2 dateOfIssue = do
   now <- getCurrentTime
-  configs <- asks (.driverOnboardingConfigs)
+  onboardingDocumentConfig <- QODC.findByMerchantIdAndDocumentType person.merchantId DL >>= fromMaybeM (MerchantServiceUsageConfigNotFound person.merchantId.getId)
   let imageExtractionValidation =
-        if isNothing dateOfIssue && configs.checkImageExtraction
+        if isNothing dateOfIssue && onboardingDocumentConfig.checkExtraction
```

Comment:
why rename of field?

Reply:
@0utkarsh I have asked him to rename. We don't need to specify image.